src/transformers/Streamable.js

```diff
@@ -1,6 +1,13 @@
 import { URL } from 'url';
 import fetch from 'node-fetch';
 
+export const getNormalizedStreamableUrl = url => {
+  const splitUrlSegments = url.split('/');
```

Comment:
I think doing it like this, will make the meaning of the code less descriptive and more declaring what we're doing in the code.

```js
export const getNormalizedStreamableUrl = url => {
  const { pathname } = new URL(url);
  const trimmedPathName = getTrimmedPathName(pathname).split('/');

  if (trimmedPathName.length === 1) {
    return url;
  }

  return `https://streamable.com/${trimmedPathName[1]}`;
};
```